eager2-core/build.rs

```diff
@@ -0,0 +1,201 @@
+use std::{
```

Comment:
I'm not willing to put build scripts in `core`. I'd like to avoid them as much as possible in both crates (although I suspect `cfg` is gonna have to have some), but especially in `core`, since doing so really hurts build times which  discourages use of this crate.